inc/functions.inc

```diff
@@ -32800,9 +32800,13 @@ if (!function_exists('pupesoft_log')) {
 }
 
 if (!function_exists('tuoteperheen_lapsien_kertoimet')) {
-  function tuoteperheen_lapsien_kertoimet($isa_tuoteno, $tyyppi = 'P') {
+  function tuoteperheen_lapsien_kertoimet($params) {
     global $yhtiorow, $kukarow;
 
+    $isa_tuoteno = isset($params['isa_tuoteno']) ? $params['isa_tuoteno'] : die('Is�tuoteno pit�� antaa');
```

Comment:
@tiere Onko die(); oikea tapa tässä? Kuoleeko skripti rumasti vai kuuluuko olla just näin?